_posts/en/2019-01-20-rbf-in-the-wild.md

```diff
@@ -0,0 +1,243 @@
+
+GreenAddress has two places in the UI where the original and replacement transaction are associated. On the left, the transaction details screen  of the replacement transaction notes a “Double spend by txhash” field which is the transaction ID of the original transaction. The screenshot on the right of the transaction list shows an expandable section which lists any original transactions that have been replaced by this replacement transaction.
+
+### Samourai: Credit Balance of Original Transaction
+
+![Samourai wallet transaction list before and after balance update screenshots](/img/posts/rbf-in-the-wild/rbf-samourai-credit-balance-with-rbf.png)
+
+Samourai wallet shows both the original and replacement transaction with no RBF label. Additionally the balance was actually credited for both transactions. Within an hour, and before the replacement transaction confirmed, the original transaction was discarded and the correct balance appeared (right image).
+
+A user seeing two normal looking transactions and an increased balance might be confused when the transaction later disappears and their balance is smaller.
+
+### Xapo: Email Notifications for Unconfirmed Txs
+
+![Xapo emails showing incoming transactions screenshot](/img/posts/rbf-in-the-wild/rbf-xapo-email-notifications.png)
+
+Xapo sends email confirmations for both the original and replacement transaction which might indicate to a receiver that they will get 2,000 bits total via two transactions, instead of the actual 1,000 bits total in one transaction in this example.
+
+### Ledger Live: Error
+
+![Ledger Live's Portfolio showing red error message screenshot](/img/posts/rbf-in-the-wild/rbf-ledger-dashboard-sync-error-message.png)
+
+Ledger Live’s internal database was corrupted by the replacement transaction.
+
+Note: The fix was to reinstall Ledger Live. Optech is in communication with Ledger regarding this issue.
+
+### Electrum: Advanced Control of RBF Fee Bump
+
+![Electrums dialog of advanced fee bumping options including slider and textbox screenshot](/img/posts/rbf-in-the-wild/rbf-electrum-advanced-rbf-options.png)
+
+Electrum allows more advanced control of the fees in the UI. Additionally the dialog provides the capability to mark the transaction as final.
+
+### BRD: Generic Error Messages
+
+![BRD wallet transactions list screen showing a failed transaction screenshot](/img/posts/rbf-in-the-wild/rbf-brd-transaction-list-incoming-bumped.png)
+
+BRD shows a “Failed” label on the replacement transaction. In testing, the “Failed” transaction actually ends up being the transaction that confirms.
+
+## Usability Considerations
+
+A blanket recommendation for wallets, explorers, and other Bitcoin services would be to support Opt-in RBF, BIP125. Most of the services tested in this study do not have any support for RBF. As congestion builds on the network in the future, RBF will be an important tool in the scaling toolbox.
```

Comment:
Nit: remove ', BIP125' here. You've already established what you mean by opt-in RBF at the top.

Reply:
Good catch.